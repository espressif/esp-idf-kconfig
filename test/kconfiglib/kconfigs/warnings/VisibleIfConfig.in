mainmenu "Test if visible if for config and choice raises warning"

    config FUEL
        visible if MOTORS_ENABLED
        string "Fuel"
        default "kerosene"

    choice LUNCH
        visible if CREW
        prompt "Lunch"

        config LUNCH_SANDWICH
            bool "Sandwich"

        config LUNCH_PIZZA
            bool "Pizza"

        config LUNCH_SALAD
            bool "Salad"
    endchoice
