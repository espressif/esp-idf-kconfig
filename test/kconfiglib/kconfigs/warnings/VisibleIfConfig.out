#
# Automatically generated file. DO NOT EDIT.
# Espressif IoT Development Framework (ESP-IDF)  Project Configuration
#
# default:
CONFIG_FUEL="kerosene"
# default:
CONFIG_LUNCH_SANDWICH=y
# default:
# CONFIG_LUNCH_PIZZA is not set
# default:
# CONFIG_LUNCH_SALAD is not set
